packages/amplify-cli/tests/amplify-helpers/headless-init.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+IFS='|'
+PROJECTNAME=\"headlessProjectName\"
+EDITOR=\"code\"
+FRONTEND=\"javascript\"
+PROVIDERS="[\"awscloudformation\"]"
+AMPLIFY="{\
+\"projectName\":$PROJECTNAME,\
+\"defaultEditor\":$EDITOR,\
+\"providers\":$PROVIDERS,\
+\"frontend\":$FRONTEND\
+}"
+REACTCONFIG="{\
+\"SourceDir\":\"src\",\
+\"DistributionDir\":\"build\",\
+\"BuildCommand\":\"npm run-script build\",\
+\"StartCommand\":\"npm run-script start\"\
+}"
+JAVASCRIPT="{\
+\"framework\":\"react\",\
+\"config\":$REACTCONFIG\
+}"
+AWSCLOUDFORMATIONPPROJECTCONFIG="{\
+\"useProfile\":true,\
+\"profileName\":\"default\",\
+\"accessKeyId\":\"headlessaccesskeyid\",\
+\"secretAccessKey\":\"headlesssecrectaccesskey\",\
+\"region\":\"us-east-1\"\
+}"
+AWSCLOUDFORMATION="{\
+\"configLevel\":\"project\",\
+\"config\":$AWSCLOUDFORMATIONPPROJECTCONFIG\
+}"
+amplify init \
```

Comment:
Talked to @mlabieniec @yuth   about this as well to get their opinions on it, and I think we should have the command something like this. (the same goes for the configure command).

```
amplify init \
--amplify $AMPLIFY \
--frontend $FRONTEND_CONFIG \
--provider $PROVIDER_CONFIG \
```

```
FRONTEND_CONFIG="{\
\"frontend\":\"javascript\",\
\"framework\":\"react\",\
\"config\":$REACTCONFIG\
}"

```

```
PROVIDER_CONFIG="{\
\"awscloudformation\": {
  \"useProfile\":true,\
  \"profileName\":\"default\",\
  \"accessKeyId\":\"headlessaccesskeyid\",\
  \"secretAccessKey\":\"headlesssecrectaccesskey\",\
   \"region\":\"us-east-1\"\
 },
}"

```

Reply:
Good idea, this gives the user a cleaner CLI interface